pkg/chunkenc/gzip.go

```diff
@@ -329,11 +348,25 @@ func (c *MemChunk) UncompressedSize() int {
 	return size
 }
 
-// Utilization implements Chunk.  It is the bytes used as a percentage of the
-func (c *MemChunk) Utilization() float64 {
-	size := c.UncompressedSize()
+// CompressedSize implements Chunk
+func (c *MemChunk) CompressedSize() int {
+	size := 0
+	// Better to account for any uncompressed data than ignore it even though this isn't accurate.
+	if !c.head.isEmpty() {
+		size += c.head.size
+	}
+	size += c.cutBlockSize
+	return size
+}
 
-	return float64(size) / float64(blocksPerChunk*c.blockSize)
+// Utilization implements Chunk.
+func (c *MemChunk) Utilization() float64 {
+	if c.targetSize != 0 {
+		return float64(c.targetSize) / float64(c.CompressedSize())
```

Comment:
I'm not sure this is right, shouldn't it be `compressed_size / target_size`?

Reply:
and that else though.

Reply:
compresses size = 500kb
target size = 1000kb

Current Method Utilization = 1000/500 = 2... You win this round @rfratto